README.md

```diff
@@ -3,6 +3,19 @@
 
 > official website for opensource254
 
+## Reguirements for Debian
```

Comment:
It is fine though this is not the purpose of this readme.
This README.MD Assumes that you have node and npm in your system

Reply:
the reason ni node ya kitambo inaleta errors

Reply:
> the reason ni node ya kitambo inaleta errors

Yes, maybe we need to add this application needs version > 8. Then create a gist on how to install node on Debian and link it there.

Imagine if we have to put the instructions of node installation on all systems.

Reply:
hehe sounds good

Reply:
Hehe have you made the requested changes?